lib/tailwind/src/components/templates/booleanTemplate.tsx

```diff
@@ -0,0 +1,38 @@
+import z from "zod";
+import { useErrorsAtPath, useFormDataAtPath } from "../..";
+import { ErrorsListTemplate } from "./errorsListTemplate";
+import React from "react";
+
+export const BooleanTemplate: React.FC<{
+  schema: z.ZodBoolean;
+  path: string[];
+  liveValidate?: boolean;
+}> = ({ schema, path, liveValidate }) => {
+  const [value, setValue] = useFormDataAtPath(path);
+  const [errors, setErrorsAtPath] = useErrorsAtPath(path);
+  const onBlur = (event: React.FocusEvent<HTMLInputElement>) => {
+    if (liveValidate) {
+      try {
+        schema.parse(event.target.value);
+        setErrorsAtPath([]);
+      } catch (validationErrors) {
+        const errors = validationErrors as z.ZodError;
```

Comment:
```
const errors: z.ZodError = validationErrors;
````

Reply:
TS2322: Type unknown is not assignable to type ZodError<any>